requirements-dev.txt

```diff
@@ -75,124 +93,24 @@ rsa==4.9; python_version >= '3.6' and python_version < '4'
 s3transfer==0.7.0; python_version >= '3.7'
 sasl==0.3.1
 setuptools==68.2.2; python_version >= '3.8'
+setuptools-git==1.2
 six==1.16.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 sqlalchemy==1.4.49; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5'
 sqlalchemy-bigquery==1.8.0; python_version < '3.12' and python_version >= '3.7'
 sqlalchemy-redshift==0.8.14; python_version >= '3.4'
 sqlalchemy-trino==0.5.0; python_version >= '3.7'
+stanio==0.3.0; python_version >= '3.8'
 tenacity==8.2.3; python_version >= '3.7'
 thrift==0.16.0; python_version >= '3.0'
 thrift-sasl==0.4.3
+tqdm==4.66.1; python_version >= '3.7'
 trino[sqlalchemy]==0.327.0; python_version >= '3.7'
+tzdata==2023.3; python_version >= '2'
 tzlocal==5.2; python_version >= '3.8'
 urllib3[socks]==1.26.17; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5'
 werkzeug==2.3.7; python_version >= '3.8'
+wheel==0.41.3; python_version >= '3.7'
 zipp==3.17.0; python_version >= '3.8'
 zope.event==5.0; python_version >= '3.7'
 zope.interface==6.1; python_version >= '3.7'
 zstd==1.5.5.1
-alembic==1.12.1; python_version >= '3.7'
```

Comment:
`requirements-dev.txt` seemed to contain both default and dev dependencies, I've regenerated it with a `--dev-only` option, so now it contains only the dev dependencies.